scripts/version-rke2

```diff
@@ -1 +1 @@
-RKE2_VERSION="v1.22.12+rke2r1"
+RKE2_VERSION="v1.23.11-rc1+rke2r1"
```

Comment:
Q: do we plan to bump to a stable `rke2+v1.23.11` version before the v1.1.0 release?

Reply:
Need to be v1.23.12 due to a regression.
I'd like to first stick to v1.23.10 first. I'll update the PR.